english/dream/dream.md

```diff
@@ -5,8 +5,6 @@ You start to feel excted about the possibilities of this unique world.
 You feel that there will be no consequence in your dream and you attempt to manipulate your dream away, but how...
 Afterwards, you decide to:
 
-[scream](../scream/scream.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.